crates/scheduler/src/call/led.rs

```diff
@@ -44,8 +44,10 @@ fn count<B: Board>(call: SchedulerCall<B, api::count::Sig>) {
 fn get<B: Board>(call: SchedulerCall<B, api::get::Sig>) {
     let api::get::Params { led } = call.read();
     let result = try {
-        let id = Id::new(*led as usize).ok_or(Trap)?;
-        board::Led::<B>::get(id)
+        match Id::new(*led as usize) {
+            Some(id) => board::Led::<B>::get(id),
+            None => Err(Error::new(Space::User, Code::InvalidArgument)),
```

Comment:
nit: if I remember correctly there's a shorthand for that
```suggestion
            None => Err(Error::user(Code::InvalidArgument)),
```

Reply:
Done.